devnet/cmd/scale-down-ecs-service/main.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"regexp"
+	"syscall"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/jessevdk/go-flags"
+)
+
+var (
+	svc *ecs.ECS
+)
+
+func init() {
+	sess := session.Must(session.NewSessionWithOptions(session.Options{
+		SharedConfigState: session.SharedConfigEnable,
+	}))
+	svc = ecs.New(sess)
+}
+
+func findServiceArns(ctx context.Context, cluster, serviceRegex string) (serviceArns []*string, err error) {
+	r, err := regexp.Compile(serviceRegex)
+	if err != nil {
+		return
+	}
+
+	var lsi = &ecs.ListServicesInput{
+		Cluster: &cluster,
+	}
+	for {
+		var lso *ecs.ListServicesOutput
+		lso, err = svc.ListServicesWithContext(ctx, lsi)
+		if err != nil {
+			return
+		}
+		for _, arn := range lso.ServiceArns {
+			if r.MatchString(*arn) {
+				serviceArns = append(serviceArns, arn)
+			}
+		}
+		if lso.NextToken != nil {
+			lsi.NextToken = lso.NextToken
+		} else {
+			break
+		}
+	}
+	return
+}
+
+func updateServices(ctx context.Context, cluster string, serviceArns []*string) (err error) {
```

Comment:
This function could be renamed to `scaleTo` and accept the `desiredCount` as an argument, or just rename it to `scaleToZero`